src/index.html

```diff
@@ -1,14 +1,117 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Dia</title>
-    <link rel="stylesheet" href="./styles/main.css">
-  </head>
-  <body>
-    <img src="./images/logo.svg" alt="Logo">
-    <h1>Dia</h1>
-    <script type="text/javascript" src="scripts/main.js"></script>
-  </body>
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Dia</title>
+  <link rel="icon" href="images/logo/favicon_new.png">
+  <link rel="stylesheet" href="./styles/main.css">
+  <link href="https://fonts.googleapis.com/css?family=Poppins:400,500,600,700&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,500,600,700&display=swap" rel="stylesheet">
+</head>
+
+<body>
+  <header class="header">
+    <img src="images/logo/Logo.svg" alt="Logo" class="header__logo">
+    <div id="menu-bar">
+      <div id="menu" onclick="onClickMenu()">
+        <div class="bar bar__first"></div>
+        <div class="bar bar__second"></div>
+        <div class="bar bar__third"></div>
+      </div>
+    </div>
+    <div class="menu-bg" id="menu-bg"></div>
+    <nav class="header__nav nav">
```

Comment:
спочатку блок, потім елемент
```suggestion
    <nav class="nav header__nav">
```

Reply:
Юра сказав на уроці це не принципово, мені так просто зручніше орієнтуватись по верстці